docs/index.rst

```diff
@@ -148,6 +95,11 @@ Pyramid
 
 .. automodule:: ddtrace.contrib.pyramid
 
+Aiohttp
```

Comment:
nitpick but I don't think those should be capitalized if the project names are always displayed lower case "aiohttp", "gevent" ...